spec/migrator/alter_table_statement_spec.cr

```diff
@@ -19,25 +19,25 @@ describe Avram::Migrator::AlterTableStatement do
       remove_belongs_to :employee
     end
 
-    built.statements.size.should eq 7
     built.statements.first.should eq <<-SQL
     ALTER TABLE users
       ADD name text,
-      ADD email text,
+      ADD email text NOT NULL,
       ADD nickname text NOT NULL,
       ADD age int NOT NULL DEFAULT 1,
       ADD num bigint NOT NULL DEFAULT 1,
       ADD amount_paid decimal(10,5) NOT NULL DEFAULT 1.0,
       ADD completed boolean NOT NULL DEFAULT false,
       ADD meta jsonb NOT NULL DEFAULT '{"default":"value"}',
       ADD joined_at timestamptz NOT NULL DEFAULT NOW(),
-      ADD updated_at timestamptz,
+      ADD updated_at timestamptz NOT NULL,
```

Comment:
This refactor actually caught a bug in the alter statement. These should have been NOT NULL from the start